### PR TITLE
Improve Warning Dialog by Invalid/Incomplte Model

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- Add icon to dialog:
   - Yellow Warning icon - incomplete file
   - Red warning icon - invalid file
- Replace _new_ buttons in toolbar with warning icon button

### CODE REVIEW

File: java/bundles/org.eclipse.set.application/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -47,4 +47,9 @@ TableMenuService_Table=Zeige in Tabelle
...
 ControlAreaCombo_All_Objects_Value=Gesamter Planungsinhalt
 ControlAreaCombo_Default_Area_Name=Stellbereich %d
-ControlAreaCombo_PlanningArea_Value=Planungsbereich
\ No newline at end of file
+ControlAreaCombo_PlanningArea_Value=Planungsbereich
+
+IncompleteFileDialog_Title=Unvöllständige PlanPro-Datei
+IncompleteFileDialog_Message=Die geladende Datei ist für den Anwendungsfall PT 1 nicht vollständig. Es kann fehlerhaften oder unvollständigen Darstellungen kommen.
+InvalidFileDialog_Title=Ungültiges PlanPro-Modell
+InvalidFileDialog_Message=Die geladende Datei enthält kein valides Modell im PlanPro-Format. Es kann unvorhergesehenen Fehlern und Abstürzen kommen.
```
Comment (mariusheine) [inline java/bundles/org.eclipse.set.application/OSGI-INF/l10n/bundle.properties:55]: @TruongQuangSB Just some grammar and typo stuff

```suggestion
IncompleteFileDialog_Message=Die geladene Datei ist für den Anwendungsfall PT 1 nicht vollständig. Es kann zu fehlerhaften oder unvollständigen Darstellungen kommen.
InvalidFileDialog_Title=Ungültiges PlanPro-Modell
InvalidFileDialog_Message=Die geladene Datei enthält kein valides Modell im PlanPro-Format. Es kann zu unvorhergesehenen Fehlern und Abstürzen kommen.
```

File: java/bundles/org.eclipse.set.core.services/src/org/eclipse/set/core/services/dialog/DialogService.java [hunk 2]
```diff
@@ -235,6 +235,21 @@ void error(Shell shell, String title, String message, String technicalMsg,
...
+	 * @param dialogTitle
+	 *            the dialog title
+	 * @param dialogMsg
+	 *            the message
+	 * @param buttonsLabel
+	 *            the buttons label
+	 * @return whether the model should be loaded (<code>true</code>) or whether
+	 *         the loading should be aborted (<code>false</code>)
+	 */
+	boolean loadIncomleteModel(Shell shell, String dialogTitle,
```
Comment (mariusheine) [inline java/bundles/org.eclipse.set.core.services/src/org/eclipse/set/core/services/dialog/DialogService.java:250]:
```suggestion
	boolean loadIncompleteModel(Shell shell, String dialogTitle,
```

File: java/bundles/org.eclipse.set.application/src/org/eclipse/set/application/Messages.java [hunk 3]
```diff
@@ -190,4 +190,26 @@ public class Messages {
...
+
+	/**
+	 * Ungültiges PlanPro-Modell
+	 */
+	public String InvalidFileDialog_Title;
+
+	/**
+	 * Die geladende Datei enthält kein valides Modell im PlanPro-Format. Es
+	 * kann unvorhergesehenen Fehlern und Abstürzen kommen.
+	 */
```
Comment (mariusheine) [inline java/bundles/org.eclipse.set.application/src/org/eclipse/set/application/Messages.java:213]: @TruongQuangSB Same typos as above in `l10n/bundle.properties`

File: java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/dialogservice/InformationDialogWithIcon.java
Comment (mariusheine) [inline java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/dialogservice/InformationDialogWithIcon.java:1]: @TruongQuangSB This class is not used anywhere

**Review by mariusheine — CHANGES_REQUESTED** [review]

File: java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/dialogservice/InformationDialogWithIcon.java
Comment (TruongQuangSB) [reply java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/dialogservice/InformationDialogWithIcon.java:1]: ups, i think, i was remove it 😄

File: java/bundles/org.eclipse.set.utils/icons/warning_transparent.xcf
[the remark this answers is not in the document]
Comment (TruongQuangSB) [reply java/bundles/org.eclipse.set.utils/icons/warning_transparent.xcf:1]: no, we dont need it, i have forgot to remove

File: java/bundles/org.eclipse.set.application/src/org/eclipse/set/application/Messages.java (hunk 3 above)
Comment (TruongQuangSB) [reply java/bundles/org.eclipse.set.application/src/org/eclipse/set/application/Messages.java]: ok

**Review by mariusheine — APPROVED** [review]

**Pull request merged**